.github/workflows/ci-pr-only.yml

```diff
@@ -101,3 +101,45 @@ jobs:
         id: pre-commit
         shell: bash
         run: ./gitlab-ci/src/ci-scripts/pre-commit.sh
+  dependencies-check:
```

Comment:
@venkkatesh-sekar, you need to define these under `.github/workflows-source`. We have that due to DRY reasons.

Reply:
Thanks @marko-k0, I moved them